### ISSUE TITLE
[Feature request] Provide the ability to display the entire chain in the notification

### ISSUE THREAD

**Issue by Cycyfyfyf** [guardianproject/orbot#841]
Please provide the ability to display the entire chain of connections in the advanced notification:

- Input node
- Middle node
- Output node

This was already done in earlier versions of the application, but unfortunately this feature is broken at the moment :(

**Comment by Gftftf** [comment]
@n8fr8 can this be fixed? I see that that worked fine in really old versions…

**Comment by n8fr8** [comment]
There was an issue with how often we were querying for this data... essentially automatically on every circuit. We need to change it to only when the user wants to see it. This is how we have implemented it in OnionBrowser for iOS. 

So yes, we can and are thinking about how to implement this again in a safe, scalalable way.

**Comment by Yfyctc** [comment]
> So yes, we can and are thinking about how to implement this again in a safe, scalalable way.

I suggest another option. Move this information from the notification to a separate item in the app itself.

That is, display this information only after pressing the corresponding button in the application itself.
